ludwig/schema/__init__.py

```diff
@@ -68,7 +69,6 @@ def get_schema(model_type: str = MODEL_ECD):
     return schema
 
 
-@DeveloperAPI
 @lru_cache(maxsize=2)
 def get_validator():
```

Comment:
Are you planning on moving this over as well?

Reply:
Moving this isn't strictly necessary. The JSON validator can continue to live here, if it pleases. WDYT?

Reply:
It will only ever be created by `validate_config` so it should live alongside that function IMO